wls-gui-admintool/tests/types/wahltag/WahltagEvent.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { useWahltagTestDataFactory } from "@tests/types/wahltag/WahltagTestDataFactory.ts";
+import { describe, expect, it } from "vitest";
+
+import { compareByNummerAsc } from "@/types/wahltag/WahltagEvent";
+
+const { prepareWahltagEvent } = useWahltagTestDataFactory();
+
+describe("WahltagEvent.ts", () => {
+  describe("compareByNummerAsc", () => {
+    it("should_return0_when_bothNummersAreEqual", () => {
```

Comment:
```suggestion
    it("should_return0_when_bothNumbersAreEqual", () => {
```
bei den anderen auch bitte anpassen

Reply:
Das hatte ich überlegt. Ich habe mich für `Nummer` entschieden weil die Property so heißt (die nicht mal eine Zahl ist ...)

Reply:
dann evtl `bothNummerAttributes` ?

Reply:
fixed mit 2827961a4e332b62c5b5698d1e33fe915f9c1b39